src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -84,7 +94,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         $filenames = (array) $input->getArgument('filename');
+        $config = $input->getOption('config');
+        $excludes = $input->getOption('exclude');
         $this->format = $input->getOption('format');
+        $flags = $input->getOption('parse-tags');
+
+        if ($config) {
+            if (!file_exists($config)) {
+                throw new RuntimeException(sprintf('Lint config "%s" not found.', $config));
+            }
+
+            $lintConfig = $this->parseLintConfig($config);
+
+            if (null === $this->format && isset($lintConfig['format'])) {
+                $this->format = $lintConfig['format'];
+            }
+            if (false === $flags && isset($lintConfig['parse-tags'])) {
+                $flags = $lintConfig['parse-tags'];
+            }
+            if (!\count($filenames) && isset($lintConfig['includes'])) {
+                $filenames = $lintConfig['includes'];
```

Comment:
I think I would merge includes from config file and CLI options.

Reply:
OK changed.